04/app.js

```diff
@@ -1 +1,21 @@
-const years = [1980, 1934, 2002, 2019];
\ No newline at end of file
+const years = [1980, 1934, 2002, 2019];
+const currYear = new Date().getFullYear();
+
+// SPOSÓB 1
+// const passedYears = years.map(countDiff);
+
+// function countDiff(year) {
+//   return currYear - year    
+// }
+
+// console.log(passedYears)
+// ------------------------------
+
+// 2 SPOSÓB - czyżby trochę lepszy z tej racji, że nie musimy za każdym razem stosować map? :)
+
+const passedYears = countDiff(years)
+
+function countDiff(arr) {
+  return arr.map(function(year) {return currYear - year})};
+
+console.log(passedYears)
```

Comment:
Ja bym połączył oba rozwiązania, zrobił bym 2 funkcje
1) oblicza różnice dla 1 wartości (u ciebie to sposób 1)
2) zwraca tablice z różnicami czyli wew. mamy map(), który korzysta z funkcji 1